package.json

```diff
@@ -45,6 +50,7 @@
     "html"
   ],
   "devDependencies": {
+    "@arethetypeswrong/cli": "^0.12.2",
```

Comment:
Is this dependency used?

Reply:
Here (line 95): https://github.com/markedjs/marked/pull/3038/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R95

It’s optional, but I recommend it any time types and JS are generated by separate tools, as opposed to a single `tsc` run. As evidenced by the issue this PR fixes, it is quite easy to make JS that disagrees with the types when using a tool like Rollup.

Reply:
Oh I see, thats the `attw` bin 👍 

Oddly enough, `@arethetypeswrong/cli` depends on `marked` 😆 

https://www.npmjs.com/package/@arethetypeswrong/cli?activeTab=dependencies

Reply:
That’s how I noticed this issue! 😄